packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -343,7 +343,7 @@ public Future<DocumentAsset> extractPDF(final UUID documentId, final String doma
 				notificationInterface.sendFinalMessage("Extraction complete");
 
 				// return the final document
-				return documentService.updateAsset(document).orElseThrow();
+				return documentService.getAsset(documentId).orElseThrow();
```

Comment:
We still need to update the Asset with the new metadata field, or I'm missing something?
@dvince2 @kbirk

Reply:
🤔 that make sense now.